frontend/src/metabase-lib/parameters/utils/targets.unit.spec.ts

```diff
@@ -143,16 +143,20 @@ describe("parameters/utils/targets", () => {
       ["field", PRODUCTS.CATEGORY, null],
     ] as ParameterDimensionTarget;
 
-    it("should return null when given a card without a `dataset_query`", () => {
+    it("should return the field that maps to the mapping target event given a card without a `dataset_query` (metabase#20656)", () => {
```

Comment:
Adjust the unit test to match the new behavior. Please see https://github.com/metabase/metabase/pull/33632/files#r1315648167